recipe/meta.yaml

```diff
@@ -1,28 +1,31 @@
 {% set name = "coffea" %}
 {% set version = "2024.10.0" %}
+{% set python_min = "3.9" %}
+# v2024.10.0 supports Python 3.8, but the required dependencies are missing
+# Python 3.8 support on conda-forge.
```

Comment:
Example:

```console
$ docker run --rm -ti python:3.8 bash
root@1ce9c68a2081:/# python -m venv venv && . venv/bin/activate && python -m pip install uv && uv pip install 'coffea==2024.10.0'
(venv) root@1ce9c68a2081:/# uv pip show coffea
Using Python 3.8.20 environment at /venv
Name: coffea
Version: 2024.10.0
Location: /venv/lib/python3.8/site-packages
Requires: aiohttp, awkward, cachetools, cloudpickle, correctionlib, dask, dask-awkward, dask-histogram, fsspec-xrootd, hist, lz4, matplotlib, mplhep, numba, numpy, packaging, pandas, pyarrow, requests, scipy, toml, tqdm, uproot, vector
Required-by:
```

```console
$ docker run --rm -ti python:3.8 bash
root@3711dc87e0d3:/# curl -fsSL https://pixi.sh/install.sh | bash && . /root/.bashrc
This script will automatically download and install Pixi (latest) for you.
Getting it from this url: https://github.com/prefix-dev/pixi/releases/latest/download/pixi-x86_64-unknown-linux-musl.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 14.9M  100 14.9M    0     0  4525k      0  0:00:03  0:00:03 --:--:-- 5924k
The 'pixi' binary is installed into '/root/.pixi/bin'
Updating '/root/.bashrc'
Please restart or source your shell.
root@3711dc87e0d3:/# mkdir example && cd $_
root@3711dc87e0d3:/example# pixi init
✔ Created /example/pixi.toml
root@3711dc87e0d3:/example# pixi add 'python=3.8' 'coffea==2024.10.0'
  × failed to solve the conda requirements of 'default' 'linux-64'
  ╰─▶ Cannot solve the request because of: coffea ==2024.10.0 cannot be installed because there are no viable options:
      └─ coffea 2024.10.0 would require
         └─ dask-core >=2024.3.0, which cannot be installed because there are no viable options:
            └─ dask-core 2024.3.0 | 2024.3.1 | 2024.4.0 | 2024.4.1 | 2024.4.2 | 2024.5.0 | 2024.5.1 | 2024.5.2 | 2024.6.0 | 2024.6.1 | 2024.6.2 | 2024.7.0 | 2024.7.1 | 2024.8.0 | 2024.8.1 | 2024.8.2 |
      2024.9.0 | 2024.9.1 | 2024.10.0 | 2024.11.0 | 2024.11.1 | 2024.11.2 would require
               └─ python >=3.9, for which no candidates were found.
      

root@3711dc87e0d3:/example#
```